src/lifecycle.mjs

```diff
@@ -140,14 +140,14 @@ export async function init(worker) {
     const strategy = finder(lifeCycleType, message.name);
     const messages = await run(strategy, lifeCycleType, "init", message.args);
     messages.worker.forEach((message) => worker.postMessage(message));
-    messages.lifecycle.forEach((message) => lch.emit("message", [message]));
+    messages.lifecycle.forEach((message) => lch.emit("message", message));
```

Comment:
I believe this was a bug introduced in the latest commit. I have reverted it.

Reply:
yes makes sense